Orion build still fails after updating to c7bb062.  Here's the traceback

```
/apps/python-3.9.2/bin/python3: error while loading shared libraries: libpython\
3.9.so.1.0: cannot open shared object file: No such file or directory
make[2]: *** [saber/src/saber/util/type_rng.F90] Error 127
make[2]: Leaving directory `/work/noaa/da/rtreadon/git/GDASApp/vader/build'
make[1]: *** [saber/src/saber/CMakeFiles/saber.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/modulefiles/GDAS/orion.lua
+++ b/modulefiles/GDAS/orion.lua
@@ -57,6 +57,9 @@ setenv('R2D2_CONFIG', '/work2/noaa/da/cmartin/GDASApp/R2D2_SHARED/config_orion.y
 
 execute{cmd="ulimit -s unlimited",modeA={"load"}}
 
+-- below is hack because of hardcoded python exe in fckit
+prepend_path("LD_LIBRARY_PATH","/apps/python-3.9.2/lib")
+
 whatis("Name: ".. pkgName)
 whatis("Version: " .. pkgVersion)
 whatis("Category: GDASApp")

